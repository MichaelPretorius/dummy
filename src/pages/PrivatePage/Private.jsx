import React from 'react';

import { Header } from '../../styles';

export const Private = () => {
  return (
    <Header center secondary>
      Private
    </Header>
  );
};

export default Private;
