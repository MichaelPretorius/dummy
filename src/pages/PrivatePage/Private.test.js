import React from 'react';
import { shallow } from 'enzyme';

import { Private } from './Private';

describe('Private component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Private />);
  });

  test('should render Private component', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
