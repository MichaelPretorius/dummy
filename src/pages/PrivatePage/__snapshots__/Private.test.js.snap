// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private component should render Private component 1`] = `
"<div>
  Private
</div>"
`;
